### ISSUE TITLE
Modification: E-Fil+ Large Dataset Multisig Process - v3.0 Single multisig entity to many approved E-Fil+ clients

### ISSUE THREAD

**Issue by kevzak** [filecoin-project/notary-governance#613]
## Issue Description
This modification is meant to mimic the original large data v3 multisig design: https://github.com/filecoin-project/notary-governance/issues/509

## Impact
This will allow the creation of a separate multisig category for E-Fil+ pilot specific clients and include the members of the E-Fil+ Notary WG.

## Proposed Solution(s)
Create a single E-Fil+ LDN multisig, with all E-Fil+ notary addresses, along with members of governance team.
Address on chain for this msig: f2ldxnlo7st7qhvdc2lp3pgxku2bfaitisrcd347q

Start this LDN with 25 PiB of DataCap (to be able to potentially support 25 client POC projects at max 1PiB each). This amount and top-up rate could scale based on usage and RKH responsiveness. However, for now there no need to utilize a bot to request top-ups from RKH, this will be determined based pilot outcomes. 

By including governance team for management of the multisig, we greatly decrease the lift to manage addresses (adding or removing notaries). 

### Multisig addresses at this time
Admin:
Galen: f1a5b5cn5xgsmui2kivzj2eryzrqmhmew4tytj3by
Deep: f14l6ugtjaqp72yfrum6mwpbmw6v6rncbqp64526y
Kevin Wray: f15hsz52t266gczi2fbyx7rrxbsngek6cqmcvpthi
Kevin Zakorchemny: f1et5q4u6kcfwzjdttjjvxtcpxhj5nqopxnlqcxry
Rahgav: f1zndwxgee6xr7zlg2k2zos6aqu64uxlgajbprdmi

Notaries:
James Hoang - PiKNiK - f1ypuqpi4xn5q7zi5at3rmdltosozifhqmrt66vhq
Pluskit - f1tgnlhtcmhwipfm7thsftxhn5k52velyjlazpvka
Wijnand Schouten - Speedium - f1krmypm4uoxxf3g7okrwtrahlmpcph3y7rbqqgfa
Coffee Cloud - f14qmuid2b6ne4342m5dk56f4rcr7y5sz4sg5fiwy
Cabrina Huang - f1a2lia2cwwekeubwo4nppt4v4vebxs2frozarz3q
Eric - ByteBase - f1yh6q3nmsg7i2sys7f7dexcuajgoweudcqj2chfi
BlockMaker - f1o3twrcpwjtpcd4q36lpq4qmy2qfbgtyy5h6tsty
Matrix Storage - f1tbxqwjxfyv7swsdin4einirlsfquv3vnmlapley
Fenbushi Capital - f1yqydpmqb5en262jpottko2kd65msajax7fi4rmq
Tinfra LLC - f1jqk7xok5kautet2knhwlg74jvcfbrqlj47kbp2i
Charles Cao - f1hlubjsdkv4wmsdadihloxgwrz3j3ernf6i3cbpy
Neo Ge - IPFSMain - f13k5zr6ovc2gjmg3lvd43ladbydhovpylcvbflpa
New Web Group - f1e77zuityhvvw6u2t6tb5qlnsegy2s67qs4lbbbq
ORIGIN Storage - f1q6bpjlqia6iemqbrdaxr2uehrhpvoju3qh4lpga
Tianji Studio - f1pszcrsciyixyuxxukkvtazcokexbn54amf7gvoq
Guazi Dynamic - f1j3u7crhjzwb2cj5mq7vodlt4o66yoyci7lhcauy
IPFS Union - f17xdri3wunqgld7dm23e4f3eqsntjakwc47xjo6i
STCould: f1jvvltduw35u6inn5tr4nfualyd42bh3vjtylgci
BingHe Web3.0 Lab - f14gme3f52prtyzk6pblogrdd6b6ivp4swc6qmesi
ND Labs -  f1yayfsv6whu3rheviucvventj3y6t542xfpb47ei
1475 - f1ofq4mngy7ggcp755pfquq2gphjjnlydolf6awtq

**Comment by kevzak** [comment]
FYI @dkkapur @raghavrmadya @Kevin-FF-USA

**Comment by kevzak** [comment]
Notaries - please confirm below if you **DO NOT** want to be included in the E-Fil+ Multisig and I will remove you.

The expectations for the E-Fil+ Pilot are listed here: https://github.com/filecoin-project/notary-governance/discussions/579#discussioncomment-3198331

TLDR: at each application approval point, two of you will be randomly assigned and notified via github (and potentially also by email/slack) and given 48 Business Day Hours (Mon-Fri) to make a comment about the application.

@jamerduhgamer
@PluskitOfficial
@cryptowizzard
@davidthoms
@xingjitansuo
@swatchliu
@MegTei
@BlockMakeronline
@kernelogic
@MatrixStorage
@Fenbushi-Filecoin
@bmcnabb25
@flyworker
@neogeweb3
@newwebgroup
@derricktan23
@liyunzhi-666
@Fatman13

**Comment by kernelogic** [comment]
Hi @kevzak I apologize but please remove me from the pilot. I'm not full-time on filecoin and lately I have been busy.

**Comment by MRJAVAZHAO** [comment]
@kevzak We want to join Multisig addresses at this time. We are here.https://github.com/filecoin-project/notary-governance/discussions/579#discussioncomment-3292134
BingHe Web3.0 Lab - f14gme3f52prtyzk6pblogrdd6b6ivp4swc6qmesi

**Comment by stcouldlisa** [comment]
@kevzak  Please don't forget us

LISA---STCould:  f1jvvltduw35u6inn5tr4nfualyd42bh3vjtylgci

**Comment by kevzak** [comment]
> @kevzak We want to join Multisig addresses at this time. We are here.[#579 (comment)](https://github.com/filecoin-project/notary-governance/discussions/579#discussioncomment-3292134) BingHe Web3.0 Lab - f14gme3f52prtyzk6pblogrdd6b6ivp4swc6qmesi

What is your slack @MRJAVAZHAO ?

**Comment by kevzak** [comment]
> 

@stcouldlisa , the name Barry is mentioned on the Notary Application. What is your connection to Barry as a notary? https://github.com/filecoin-project/notary-governance/issues/429

**Comment by stcouldlisa** [comment]
Barry is a technician in our company. He and I are colleagues. When applying for a notary public, I have not registered for GitHub. Therefore, I wrote the content and asked barry to submit information on github. You can find it on our notary application form. It can be seen that ledger is managed by me, and it can also be seen on the sign-in form of the notary meeting, all of which are attended by me on behalf of the company.

**Comment by kevzak** [comment]
Adding @stcouldlisa and @MRJAVAZHAO 
Removing @megtei

**Comment by MRJAVAZHAO** [comment]
Awesome.

**Comment by MRJAVAZHAO** [comment]
> 

BingHe Web3.0 Lab(冰河web3实验室)

**Comment by Stoibers** [comment]
Hi @kevzak 
This is new github account for NDLabs. also I have DM u in slack. 
Please count me in. 
My address is f1yayfsv6whu3rheviucvventj3y6t542xfpb47ei 
Thank you!

**Comment by fireflyHZ** [comment]
Hi @kevzak 
We are FireflyHZ , address: f1fg6jkxsr3twfnyhdlatmq36xca6sshptscds7xa
We are also very concerned about E-FIL+, but some time ago we were busy missing a lot of E-FIL+ meetings, please let us join in E-FIL+ Multisig as well. Thanks!

**Comment by kevzak** [comment]
@fireflyHZ the Notary WG has already been confirmed and finalized for the pilot.
